### PR TITLE
Lets the utility belt store a bigger varation of items

### PR DESCRIPTION
**Pull request by jughu**
hi

:cl: Hyena
tweak:Utility belts now hold more things
/:cl:

[]: # (Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding:)I think these utility tools should be atleast able to fit in utility belt since why does a circuit board fit in my pocket but not my belt?

### CODE REVIEW

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -33,7 +33,22 @@
...
 		/obj/item/weapon/extinguisher/mini,
 		/obj/item/device/radio,
-		/obj/item/clothing/gloves
+		/obj/item/clothing/gloves,
+		/obj/item/device/geiger_counter,
+		/obj/item/weapon/tank/internals/emergency_oxygen,
+		/obj/item/device/gps,
+		/obj/item/borg/upgrade/rename,
+		/obj/item/borg/upgrade/restart,
+		/obj/item/clothing/glasses/meson,
```
Comment (lzimann) [inline code/game/objects/items/weapons/storage/belt.dm]: might aswell add just the root path of glasses instead of 4 different types

**Comment by KorPhaeron** [comment]
It's a storage creep episode

**Comment by PKPenguin321** [comment]
Didn't you already try this before a long time ago and get turned down

**Comment by jughu** [comment]
@pkpenguin321 thats was the wallet thingy with the mini welder

File: code/game/objects/items/weapons/storage/belt.dm [hunk 2]
```diff
@@ -33,7 +33,19 @@
...
+		/obj/item/weapon/tank/internals/emergency_oxygen,
+		/obj/item/device/gps,
+		/obj/item/borg/upgrade/rename,
+		/obj/item/borg/upgrade/restart,
+		/obj/item/clothing/glasses,
+		/obj/item/weapon/electronics,
+		/obj/item/weapon/circuitboard/computer,
+		/obj/item/weapon/hand_labeler,
+		/obj/item/hand_labeler_refill,
+		/obj/item/weapon/holosign_creator/engineering,
```
Comment (Core0verload) [inline code/game/objects/items/weapons/storage/belt.dm]: Why limit it to engineering creator?

File: code/game/objects/items/weapons/storage/belt.dm [hunk 3]
```diff
@@ -33,7 +33,19 @@
 		/obj/item/device/analyzer,
 		/obj/item/weapon/extinguisher/mini,
 		/obj/item/device/radio,
-		/obj/item/clothing/gloves
+		/obj/item/clothing/gloves,
+		/obj/item/device/geiger_counter,
+		/obj/item/weapon/tank/internals/emergency_oxygen,
```
Comment (Core0verload) [inline code/game/objects/items/weapons/storage/belt.dm]: Wasn't there a consistency issue with tanks on the belt not working in internals?

Comment (lzimann) [inline code/game/objects/items/weapons/storage/belt.dm]: it will be the same as if they were in a backpack

**Comment by Core0verload** [comment]
How about adding common botany tools to the list? They are tools too.
```
/obj/item/device/plant_analyzer
/obj/item/weapon/cultivator
/obj/item/weapon/hatchet
/obj/item/weapon/shovel/spade
```

File: code/game/objects/items/weapons/storage/belt.dm (hunk 2 above)
Comment (jughu) [inline code/game/objects/items/weapons/storage/belt.dm]: I guess your right

**Comment by jughu** [comment]
Ho shit i forgot about botany

**Comment by jughu** [comment]
not adding nuke disk to it because i dont want to look trough 30 different storages just to find dat darn disk

File: code/game/objects/items/weapons/storage/belt.dm [hunk 4]
```diff
@@ -33,7 +33,28 @@
...
+		/obj/item/weapon/electronics,
+		/obj/item/weapon/circuitboard,
+		/obj/item/weapon/hand_labeler,
+		/obj/item/hand_labeler_refill,
+		/obj/item/weapon/holosign_creator,
+		/obj/item/weapon/stock_parts/cell,
+		/obj/item/device/plant_analyzer,
+		/obj/item/weapon/cultivator,
+		/obj/item/weapon/shovel/spade,
+		/obj/item/weapon/hatchet,
```
Comment (jughu) [inline code/game/objects/items/weapons/storage/belt.dm]: I might remove the hatchet idk how i feel about keeping a decent brute weapon in the utility belt or idk adding a botany belt later on

**Comment by Shapsy** [comment]
Toolbelts are for tools, if you want to carry random clutter, SWAT belts exist for a reason

**Comment by jughu** [comment]
Its a utlity belts and stuff like GPS and disks and other things i added are basically tools but for other jobs :thinking:

**Comment by PKPenguin321** [comment]
It's only a utility belt in the code path.

>name = "toolbelt" 

:thinking:

**Comment by XDTM** [comment]
makes sense for botany tools since they are the ones who can actually make them

**Comment by jughu** [comment]
@korphaeron if i make it only the geiger counter airlock painter and the engineer projector and botany tools (excluding the hatchet) would you agree on it?

**Comment by Core0verload** [comment]
I don't think hatchets in tool belt are that bad. You can store them in any box, or pretty much everywhere because they are tiny items, for some reason.

**Pull request closed without merging**